columnflow/tasks/framework/mixins.py

```diff
@@ -1060,8 +1061,18 @@ class MLModelMixinBase(AnalysisTask):
         description="the name of the ML model to be applied",
     )
 
+    ml_model_settings = SettingsParameter(
+        default=DotDict(),
+        description="settings passed to the init function of the ML model",
+    )
+
     exclude_params_repr_empty = {"ml_model"}
 
+    @property
+    def ml_model_repr(self):
+        """Returns a string representation of the ML model instance."""
+        return str(self.ml_model_inst)
```

Comment:
Wouldn't it be better to overwrite the `__str__` property instead?

Reply:
We are overwriting the `__str__` property of the ml_model, but I don't think we want to overwrite this property from the task itself as this is really only the representation of the ml model